website/content/docs/auth/jwt.mdx

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+page_title: JWT - Auth Method
+description: >-
+  The JWT auth method allows authentication using signed JWTs
+---
+
+# JWT auth method
+
+@include 'x509-sha1-deprecation.mdx'
+
+The `jwt` auth method can be used to authenticate with Vault using JSON Web Tokens
+([JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)). This form of authentication
+is commonly used in Workload Identity Federation (WIF) workflows for machine-to-machine
+authentication.
```

Comment:
```suggestion
Use the `jwt` authN method to authenticate to Vault with JSON Web Tokens
([JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)). JWT authentication
is commonly used in Workload Identity Federation (WIF) workflows for
machine-to-machine authentication.
```

Style correction: avoid "this" as a pronoun, write in active voice with directive statements.

Reply:
Note that I'm changing "authN" back to "auth method" since this is how they're generally referred to.